src/setupProxy.js

```diff
@@ -202,6 +202,19 @@ function stubRedirects(app) {
   });
 }
 
+function stubRelease(app) {
+  app.use((req, res, next) => {
+    const {pathname} = url.parse(req.url);
+
+    if (pathname === '/rex/release.json') {
+      const releaseFile = path.join(__dirname, 'release.development.json');
```

Comment:
this json file will become old and weird very quickly, can you construct a json response here based on the actual books config? 

or maybe since the json config will never really be used in development, since it is only ever used when the book check is missed, maybe leave the release.development.json but leave the books section empty, that way its there if someone wants to test something and add a book but it doesn't have content that will need to be updated to continue making sense